http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcBaseTest.java

```diff
@@ -181,13 +181,15 @@ protected HtmlInput loginToKeycloak(String username, String password, URI reques
                 webClient.addCookie(getCookieString(cookie), requestUri.toURL(), null);
             }
         }
+
         HtmlPage keycloakLoginPage = webClient.getPage(location);
         HtmlForm loginForm = keycloakLoginPage.getForms().get(0);
         loginForm.getInputByName(KEYCLOAK_USERNAME).setValueAttribute(username);
         loginForm.getInputByName(KEYCLOAK_PASSWORD).setValueAttribute(password);
         return loginForm.getInputByName(KEYCLOAK_LOGIN);
     }
 
+
```

Comment:
It's good practice to avoid white space changes that don't relate to the changes being made in the PR.

Reply:
Implemented up to here. Working on the others now and also trying to see why the CI build failed.